public/app/core/components/editors/DashboardPickerByID.tsx

```diff
@@ -30,7 +32,10 @@ const getDashboards = (query = '') => {
   });
 };
 
-export const DashboardPicker: FC<Props> = ({ onChange, value, width, isClearable = false, invalid, disabled }) => {
+/**
+ * @deprecated prefer using dashboard uid rather than id
+ */
+export const DashboardPickerByID: FC<Props> = ({ onChange, value, width, isClearable = false, invalid, disabled }) => {
```

Comment:
This component is used in Enterprise (I think it's the only place it's used at all, so Enterprise code will need to be updated).

Reply:
dooh --- thanks.

Reply:
https://github.com/grafana/grafana-enterprise/pull/1887